source/dem/particle_wall_linear_force.cc

```diff
@@ -61,24 +62,20 @@ ParticleWallLinearForce<dim>::ParticleWallLinearForce(
            (1 - wall_poisson_ratio * wall_poisson_ratio) +
          DBL_MIN);
 
-      this->effective_coefficient_of_restitution.insert(
-        {i,
-         2 * particle_restitution_coefficient * wall_restitution_coefficient /
-           (particle_restitution_coefficient + wall_restitution_coefficient +
-            DBL_MIN)});
-
-      this->effective_coefficient_of_friction.insert(
-        {i,
-         2 * particle_friction_coefficient * wall_friction_coefficient /
-           (particle_friction_coefficient + wall_friction_coefficient +
-            DBL_MIN)});
-
-      this->effective_coefficient_of_rolling_friction.insert(
-        {i,
-         2 * particle_rolling_friction_coefficient *
-           wall_rolling_friction_coefficient /
-           (particle_rolling_friction_coefficient +
-            wall_rolling_friction_coefficient + DBL_MIN)});
+      this->effective_coefficient_of_restitution[i] =
+        2 * particle_restitution_coefficient * wall_restitution_coefficient /
+        (particle_restitution_coefficient + wall_restitution_coefficient +
+         DBL_MIN);
+
+      this->effective_coefficient_of_friction[i] =
+        2 * particle_friction_coefficient * wall_friction_coefficient /
+        (particle_friction_coefficient + wall_friction_coefficient + DBL_MIN);
```

Comment:
Just curious, why is there DBL_MIN? Is it to avoid division by 0 so a NaN result?

Reply:
Yup